utils/src/ooinstall/install_transactions.py

```diff
@@ -126,8 +126,20 @@ def run_main_playbook(hosts, hosts_to_run_on):
         facts_env['ANSIBLE_CONFIG'] = CFG.settings['ansible_config']
     return run_ansible(main_playbook_path, inventory_file, facts_env)
 
+
 def run_ansible(playbook, inventory, env_vars):
     return subprocess.call(['ansible-playbook',
                              '--inventory-file={}'.format(inventory),
                              playbook],
                              env=env_vars)
+
+def run_uninstall_playbook():
```

Comment:
@dgoodwin When I was working on this, I had the notion that each step in the install process would be encapsulated as "transactions" that would be independent from each other except where order was important. I don't think I got very far with that concept before I had to move on to other work. :(